<launch>
  <arg name="filename" default="hemisphere_icosahedron.csv" />

  <node name="viewpoint_planner"
        pkg="linemod_3dpose_estimator" type="viewpoint_planner"
        output="screen">
    <rosparam subst_value="true">
      filename : $(find linemod_pose_estimator)/"data/$(arg filename)"
    </rosparam>
  </node>
</launch>
